src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -63,85 +93,250 @@ private void CheckForAndConvertJsonConfig()
+
+            Options.AttachStacktrace = EditorGUILayout.Toggle(
+                new GUIContent("Stacktrace For Logs", "Whether to include a stack trace for non " +
+                                                      "error events like logs. Even when Unity didn't include and no " +
+                                                      "exception was thrown. Refer to AttachStacktrace on sentry docs."),
+                Options.AttachStacktrace);
+
+            // Options.StackTraceMode = (StackTraceMode) EditorGUILayout.EnumPopup(
+            //     new GUIContent("Stacktrace Mode", "Enhanced is the default." +
+            //                                       "\n - Enhanced: Include async, return type, args,..." +
+            //                                       "\n - Original - Default .NET stack trace format."),
+            //     Options.StackTraceMode);
+
+            EditorGUILayout.Space();
+            EditorGUI.DrawRect(EditorGUILayout.GetControlRect(false, 1), Color.gray);
+            EditorGUILayout.Space();
 
-            GUILayout.Label(new GUIContent(GUIContent.none), EditorStyles.boldLabel);
+            Options.SendDefaultPii = EditorGUILayout.BeginToggleGroup(
+                new GUIContent("Send default Pii", "Whether to include default Personal Identifiable " +
+                                                   "Information."),
+                Options.SendDefaultPii);
+
+            Options.IsEnvironmentUser = EditorGUILayout.Toggle(
+                new GUIContent("Auto Set UserName", "Whether to report the 'Environment.UserName' as " +
+                                                    "the User affected in the event. Should be disabled for " +
+                                                    "Android and iOS."),
+                Options.IsEnvironmentUser);
+
+            Options.ServerNameOverride = EditorGUILayout.TextField(
+                new GUIContent("Server Name Override", "The name of the server running the application." +
+                                                       "\nThis option is an override."),
+                Options.ServerNameOverride);
+
+            EditorGUILayout.EndToggleGroup();
+
+            EditorGUILayout.Space();
+            EditorGUI.DrawRect(EditorGUILayout.GetControlRect(false, 1), Color.gray);
+            EditorGUILayout.Space();
+
+            Options.MaxBreadcrumbs = EditorGUILayout.IntField(
+                new GUIContent("Max Breadcrumbs", "Maximum number of breadcrumbs that get captured." +
+                                                  "\nDefault: 100"),
+                Options.MaxBreadcrumbs);
+
+            Options.ReportAssembliesMode = (ReportAssembliesMode)EditorGUILayout.EnumPopup(
+                new GUIContent("Report Assemblies Mode", "Whether or not to include referenced assemblies " +
+                                                         "in each event sent to sentry."),
```

Comment:
It's not just about include or not include but also about Version vs InformationalVersion
```suggestion
                new GUIContent("Report Assemblies Mode", "Whether or not to include referenced assemblies Version or InformationalVersion" +
                                                         "in each event sent to sentry."),
```